contracts/protocols/GnosisAuction.sol

```diff
@@ -73,7 +72,7 @@ library GnosisAuction {
                 // we are selling all of the otokens minus a fee taken by gnosis
                 uint96(oTokenSellAmount),
                 // the minimum we are willing to sell all the oTokens for. A discount is applied on black-scholes price
-                uint96(optionPremium.mul(oTokenSellAmount)),
+                uint96(auctionDetails.oTokenPremium.mul(oTokenSellAmount)),
```

Comment:
This is what I was referring to https://github.com/ribbon-finance/ribbon-v2/pull/11/files#r650442961